src/Views/CheckView.vala

```diff
@@ -111,11 +112,11 @@ public class Installer.CheckView : AbstractInstallerView {
             critical ("Couldn't read apt sources: %s", e.message);
         }
 
-        if (get_vm ()) {
+        if (get_vm () || Installer.App.test_mode) {
             message_box.add (vm_view);
         }
 
-        if (!minimum_specs) {
+        if (!minimum_specs || Installer.App.test_mode) {
```

Comment:
Is this change meant to added for testing this branch? This might look a little strange when someone run the installer in the test mode on real machines.

Reply:
Removed!